src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
```

Comment:
You ONLY update the the actual scroll position on the DOM node in `componentDidUpdate` you need to update it it both cDU and `componentDidMount`. That it's working now is only because the component is triggering a few updates, but its not actually setting the scroll position after the component mounts, only calculating it.